src/app/model/server/component.ts

```diff
@@ -11,6 +10,7 @@ export class SComponent {
     activelyAffectedBy: STypedReference[] = [];
     labels: Map<string, string> = new Map();
 
+    // eslint-disable-next-line @typescript-eslint/no-explicit-any
```

Comment:
This is your test code that gets replaced by the API client from [SovereignCloudStack/status-page-openapi](https://github.com/SovereignCloudStack/status-page-openapi)?

If not, cosider generating an Interface to define the type. `npx ng g i <name>`

Reply:
Yes, this will get replaced. No sense in introducing another interface now.